platform/solr/solr-factory/src/main/java/org/codice/solr/factory/impl/ConfigurationFileProxy.java

```diff
@@ -40,14 +39,21 @@
 
     public static final String DEFAULT_SOLR_CONFIG_PARENT_DIR = "etc";
 
-    public static final String SOLR_CONFIG_LOCATION_IN_BUNDLE =
-            new AbsolutePathResolver("solr/conf").getPath();
-
     public static final String DEFAULT_SOLR_DATA_PARENT_DIR =
             new AbsolutePathResolver("data/solr").getPath();
 
     public static final String CATALOG_SOLR_COLLECTION_NAME = "metacard";
 
+    protected static final List<String> SOLR_CONFIG_FILES =
+            Collections.unmodifiableList(Arrays.asList("protwords.txt",
+                    "schema.xml",
+                    "solr.xml",
```

Comment:
That solr.xml will not be used for Solr Cloud.  The zookeeper hosts passed into the Solrj client constructor will be used.